cnc_ctrl_v1/CNC_Functions.h

```diff
@@ -33,6 +33,9 @@ libraries*/
 #define YDIRECTION BACKWARD
 #define ZDIRECTION BACKWARD
 
+#define LEFT_EEPROM_ADR     5
```

Comment:
Space needed right now is 76 bytes which is closer to 100 than i want for comfort